cloud/scope/powervs_machine.go

```diff
@@ -201,6 +201,12 @@ func (m *PowerVSMachineScope) CreateMachine() (*models.PVMInstanceReference, err
 	} else if instanceReply != nil {
 		// TODO need a reasonable wrapped error.
 		return instanceReply, nil
+	} else if instanceReply == nil {
```

Comment:
Lets add a comment why we are adding this condition and when this condition will occur.